packages/test-kit/src/with-feature.ts

```diff
@@ -299,7 +301,7 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
                             name:
                                 process.env.TRACING && process.env.TRACING !== 'true'
                                     ? process.env.TRACING
-                                    : name ?? testName?.replace(/(\W+)/gi, ' ').trim().replace(/(\W+)/gi, '-'),
+                                    : name ?? (testName ? normalizeTestName(testName) : 'nameless-test'),
```

Comment:
Not sure about it, probably should keep an eye if there are occurrences of `nameless-test`